services/navigation/builtin/builtin_test.go

```diff
@@ -3,18 +3,19 @@ package builtin
 import (
 	"context"
 	"errors"
+	"math"
```

Comment:
I completely rewrote these tests as they didn't test what we were interested in testing & were dependent on the kinematic base, when the nav service actually need not be dependent on kinematic base.